### ISSUE TITLE
Single Field Search and Search by List Fail on Type AGS-Vector

### ISSUE THREAD

**Issue by chughes-lincoln** [geomoose/gm3#870]
The standard geomoose search works with type ags-vector (i.e. specify a list of fields that the user can individually search - i.e. the search specified here - https://docs.geomoose.org/3.x/howto/configure-search-fieldrequired.html). 

It fails with both the single field search and search by list examples specified here - https://docs.geomoose.org/3.x/howto/configure-search-or-fields.html. It results in no results found and throws a console error.

Console error that occurs when trying to use search by list/single field search with ags-vector:
![image](https://github.com/user-attachments/assets/564a9480-c9c9-40c6-888b-7d89ca31cfe8)

**Comment by brentfraser** [comment]
Looking at the [GeoMoose Protocol Support](https://docs.geomoose.org/3.x/protocols.html) page, I see Search is supported (or it was when I did my testing) if the Arc GIS server is configured for  Supported Operation:Query.  Is that the case for your data source?

**Comment by chughes-lincoln** [comment]
Yes, Query is enabled, and I've gotten it working for the standard search, it's just search by list and the single field search that aren't working.

**Comment by chughes-lincoln** [comment]
Not sure how much the distinctions matter. My code matches the single-search example code, though I copied the doc code and it worked just fine on a postgres dataset. 

Regarding the ags-vector issue, I'm using the same code that's working for a wfs map-source, just swapping out the map-source name and fields.

**Comment by brentfraser** [comment]
It's been a few years, but seem to recall a problem with the "ilike" parameter.   Try switching to "like".

**Comment by brentfraser** [comment]
And maybe use the network tab in the browser's debugger to examine the query sent to the server and the server's response.

**Comment by chughes-lincoln** [comment]
I'm seeing the same issues with "like". The network tab doesn't show any traffic. For the multi-field search (that works) I can see the query to the server and the response, but for the single-field and list searches the only traffic are the search templates.

**Comment by brentfraser** [comment]
Sounds like GeoMoose is having trouble constructing the query.  Looking at the [code](https://github.com/geomoose/gm3/blob/bbb23f09063a32c6908fff341e28dd6f8d4dfce0/src/gm3/query/ags.js#L27), you better not have any single quotes in your search term.  But I expect there's a bug (or lack of doc) in GeoMoose.
